homepage.php

```diff
@@ -1,73 +1,112 @@
+<?php
```

Comment:
26/08/2021:  I add a function that deliver to the login page if they was login by want to access to some link that they should login first.